crates/brainfuck_prover/src/components/memory/table.rs

```diff
@@ -196,7 +195,7 @@ impl MemoryTable {
     ///
     /// # Errors
     /// Returns [`TraceError::EmptyTrace`] if the table is empty.
-    pub fn trace_evaluation(&self) -> Result<(TraceEval, Claim), TraceError> {
+    pub fn trace_evaluation(&self) -> Result<(TraceEval, Claim<MemoryColumn>), TraceError> {
```

Comment:
I would declare a type alias for better clarity: `pub type MemoryClaim = Claim<MemoryColumn>` instead of directly using `Claim<T>`

Reply:
Yep I wanted to go further and even declare types like

```rust
MemoryTraceEvalClaim = (TraceEval, Claim<MemoryColumn>)
```

but seems like more appropriate to do in a follow up wdyt?

Reply:
mmmh I don't think that abstracting the TraceEval and the Claim in a single type would be useful, as the eval and the claim are used in different functions afterwards

yeah we can do this in a follow up pr

Reply:
yep let us do in a follow up :)